blaze-client/src/test/scala/org/http4s/client/blaze/BlazeClientBase.scala

```diff
@@ -73,10 +73,28 @@ trait BlazeClientBase extends Http4sSuite {
           case None => srv.sendError(404)
         }
 
-      override def doPost(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
-        resp.setStatus(Status.Ok.code)
-        req.getInputStream.close()
-      }
+      override def doPost(req: HttpServletRequest, resp: HttpServletResponse): Unit =
```

Comment:
I just ported this to the com.sun server so we have a path to Jetty 11, so we have a merge conflict looming.

Reply:
I noticed it, but there wasn't much I could do if I wanted to target 0.21.